specific_use_cases/DFU_work_in_progress/INTRO_annual_consumption_model.ipynb

```diff
@@ -690,12 +724,191 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e347130a-7cff-4817-9bd1-451bd6a45712",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#help(compute_hdh_cdh) # See information about the function"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "51af09c3-cfd6-404a-8cbc-71c49deb59a8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "t2 = data_to_use[\"Air Temperature at 2m\"] # Set for all data within selected DFZ zone\n",
+    "hdh, cdh = compute_hdh_cdh(t2, hdh_threshold=60, cdh_threshold=70)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f3dc25d6-d718-4629-8d13-bd2882f3e0a6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hdh"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0005f252-cbfe-407a-a822-797ea7633f41",
+   "metadata": {},
+   "source": [
+    "Now that we have computed the HDH and CDH, we can then aggregate the results across grid cells in the forecast zone like we did previously above for HDD and CDD. We will need to do this for both the HDH and CDH variables. If you would like to change the aggregation method, you can easily modify between **median, mean, min, or max**, or write your own code to compute a weighted mean here too. We will use the *median* as an example here. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e6b64873-50d7-4134-973b-de80f237198d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hdh_aggregated = hdh.median(dim=[\"x\",\"y\"])\n",
+    "cdh_aggregated = cdh.median(dim=[\"x\",\"y\"])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5c55c615-93e0-4710-8501-bf168979e33d",
+   "metadata": {},
+   "source": [
+    "### 5b) Display a month of CDH and HDH\n",
+    "Next, we'll plot specific months of the overall timeseries produced by the CDH and HDH calculation to see the trend in degree hours. We'll use a helper plotting function, and input a start and end date of a month of interest. For example, we'll look at June of 2011, but you can input any date as the start and end dates, just be sure to follow the format of: \"YYYY-MM-DD\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e05758f2-28bf-4599-99fd-4788ce8365e6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "start_date = \"2011-06-01\"\n",
+    "end_date = \"2011-06-30\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "207ab41e-7eea-42b7-b7bf-4b690157d52e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hdh_cdh_lineplot(cdh_aggregated, start_date, end_date)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1864932c-edd1-4ab1-9e4d-57b3ac74ffe8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hdh_cdh_lineplot(hdh_aggregated, start_date, end_date)"
```

Comment:
This function is not defined - I don't see it in https://github.com/cal-adapt/climakitae/pull/191 either

Reply:
@gdicecco this function should be in utils_notebook_1.py script, not climakitae - sorry, should clarify that this py file will probably need to be updated on the hub as well! (copy and paste should be fine, I imagine)

Reply:
I'm really unsure what the issue is actually - I updated utils_notebook_1.py, and can import the other functions there by name, but the hdh_cdh_lineplot one is erroring for me (`cannot import name`)? Have you tried this on the hub as well w/o issue?

Reply:
I'll take a closer look into this!

Reply:
@gdicecco - I checked on this on the hub and everything does work as desired for hdh_cdh_lineplot function. Will you confirm for me that the hdh_cdh_lineplot is in the utils_notebook_1.py again for me? Otherwise, I might suggest we try and see what is going on via Zoom

Reply:
I did not have any issue.